CMakeLists.txt

```diff
@@ -11,7 +11,11 @@ find_package(YARP 2.3.64.13 REQUIRED)
 # Finding dependencies
 find_package(Boost REQUIRED serialization system)
 find_package(Protobuf REQUIRED)
-find_package(Gazebo   REQUIRED)
+find_package(Gazebo REQUIRED)
+if (Gazebo_VERSION_MAJOR LESS 5.0)
```

Comment:
I said something stupid, as by default the version check is `SameMajorVersion` (https://bitbucket.org/osrf/gazebo/src/73cc987353ac777b6024dc1a4eac0b7602d4635e/CMakeLists.txt?at=default&fileviewer=file-view-default#CMakeLists.txt-423) that is clearly now what we want.